sections/02_intro.md

```diff
@@ -38,16 +34,16 @@ inputs is usually multiplied by some weight and combined and summarized by the
 activation function. The value of the activation function is then multiplied by
 another set of weights to produce the output `TODO: we probably need a figure
 here - I see no way that we don't include this type of description in our paper,
-despite the fact that it's been done tons of times before.` These neural
+despite the fact that it's been done tons of times before. - I'm really partial to the nature bengio revew's presentation - really hitting on the idea of learning a feature to make a non-linear problem linear` These neural
```

Comment:
I see no way to incorporate this into a static manuscript, but I really love http://playground.tensorflow.org/ for introducing non-linear classification with neural networks.  I'm not suggesting anything actionable with this comment, just leaving it on the small chance that we would want to make our own figure instead of reusing the one @brettbj suggested.